Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -2680,7 +2680,7 @@ class PropertyWrapperFontStyle final : public PropertyWrapper<std::optional<Font
 DEFINE_ALLOCATOR_WITH_HEAP_IDENTIFIER(PropertyWrapperFontStyle);
 
 DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(PropertyWrapperFontSizeAdjust);
-class PropertyWrapperFontSizeAdjust final : public PropertyWrapperGetter<FontSizeAdjust> {
+class PropertyWrapperFontSizeAdjust final : public PropertyWrapperGetter<const FontSizeAdjust&> {
```

Comment:
FontSizeAdjust is small (2 words), I think we usually pass such types by values. It is often faster to copy such small types than passing by reference.

Reply:
I'll close this patch, based on Chris's review.